### PR TITLE
Bump to 0.1.0

### LINKED ISSUE #7: Write calendar dates to NetCDF file
This can be added trivially by taking in `t_start` as an argument for `NetCDFWriter`

### PR DESCRIPTION
**Pull request by Sbozzolo**
Collect callbacks all in one place

Running atmos build: https://buildkite.com/clima/climaatmos-ci/builds/18296#_

(Failing in allocations)

Closes #7

### CODE REVIEW

File: src/dummy_writer.jl [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"""The `DummyWriter` does nothing. It can be used to temporarily disable output of a
+scheduled diagnostic. Mostly useful for testing and debugging ClimaDiagnostics.
+"""
+struct DummyWriter <: AbstractWriter end
```
Comment (charleskawczynski) [inline src/dummy_writer.jl:4]: We could define this in the test suite if it's only used for tests, but I'm fine with leaving it here if it helps with debugging

**Review by charleskawczynski — APPROVED** [review]
This looks great, much simpler, too.

`Flame graph: perf target (IO)` seems to be a real failure, but I think we can just fix that in ClimaAtmos.

File: src/netcdf_writer_coordinates.jl [hunk 2]
```diff
@@ -112,11 +115,16 @@ function target_coordinates(space, num_points) end
...
     space::S,
-    num_points,
+    num_points;
+    disable_vertical_interpolation,
 ) where {
     S <:
     Union{Spaces.CenterFiniteDifferenceSpace, Spaces.FaceFiniteDifferenceSpace},
 }
+    if disable_vertical_interpolation
+        return Array(parent(space.grid.center_local_geometry.coordinates))[:, 1]
```
Comment (charleskawczynski) [inline src/netcdf_writer_coordinates.jl:125]:
```suggestion
        return Array(parent(Fields.coordinate_field(space).z)[:, 1]
```
I think we'd better use a ClimaCore getter here instead, is this basically `Fields.coordinate_field`?

Comment (charleskawczynski) [reply src/netcdf_writer_coordinates.jl:125]: Actually, `vec(parent(Fields.coordinate_field(space).z[Fields.ColumnIndex((1,1), 1)]))` might be even more general (as it does not rely on the DataLayout's array), but I'm not sure if this (or the existing implementation) is valid with topography

File: src/dummy_writer.jl (hunk 1 above)
Comment (Sbozzolo) [reply src/dummy_writer.jl:4]: It is used only the tests, but I personally find it useful while debugging. It is a very simple way to disable output while keeping compute active.

**Pull request merged**